composer.json

```diff
@@ -4,11 +4,19 @@
     "license": "MIT",
     "require" : {
         "php" : ">= 7.1",
-        "guzzlehttp/guzzle": "6.*",
         "ralouphie/getallheaders": "3.*",
+        "psr/http-client": "^1.0",
+        "psr/http-factory": "^1.0",
+        "php-http/discovery": "^1.7",
         "ext-curl" : "*",
         "ext-json": "*"
     },
+    "require-dev": {
+        "roave/security-advisories": "dev-master",
+        "phpunit/phpunit": "7.*",
```

Comment:
Please remove the phpunit dependency. I've switched to using phive to install dev tools. See the [contributing guide](https://github.com/nipwaayoni/elastic-apm-php-agent/blob/master/.github/CONTRIBUTING.md). I just added a note about using phpunit with an IDE.